blueman/plugins/applet/TransferService.py

```diff
@@ -171,6 +161,32 @@ def on_unload(self):
 
         self._unregister_agent()
 
+    def _make_share_path(self):
+        config_path = self._config["shared-path"]
+        default_path = GLib.get_user_special_dir(GLib.UserDirectory.DIRECTORY_DOWNLOAD)
+        path = None
+        error = False
+
+        if config_path == '':
```

Comment:
Is this true for the initial configuration or will querying `shared-path` give us `None` instead? :thinking:

Reply:
Yes, the default in the schema is an [empty string](https://github.com/blueman-project/blueman/blob/master/data/org.blueman.gschema.xml#L159).